xrdsst/controllers/init.py

```diff
@@ -62,33 +43,6 @@ def initialize_server(self, config_file):
             else:
                 self.init_security_server(configuration, security_server)
 
-    @staticmethod
```

Comment:
Due to the refacto of InitServerController, where you moved some base functionality to BaseController, there also now is a need for unit tests for BaseController. Fortunately the tests already exist, please move the unit tests "test_load_config", "test_load_config_exception", "test_init_logging", "test_init_logging_exception" and "test_initialize_basic_conf_values" into the new TestBase class, where you put the unit tests for testing BaseController.

Reply:
Commit e16e72764934eb590cc8f588dc7ef5ecd2efd9f6